terminus-ui/cohort-date-range/src/cohort-date-range.component.ts

```diff
@@ -0,0 +1,152 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  ViewEncapsulation,
+} from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+
+/**
+ * Represents date cohort object that passed in
+ */
+export interface TsDateCohort {
+  display: string;
+  range: {
+    startDate: string | Date;
+    endDate: string | Date;
+  };
+}
+
+/**
+ * Event object emitted by TsCohortDateRange when there is date range change
+ */
+
+export class TsCohortDateRangeChanged {
```

Comment:
Unneeded line break (pretty sure our docs generator won't see that these items are related if an extra line break exists)